### PR TITLE
Auto add changelog and backport labels to dependabot 

### PR DESCRIPTION
**Pull request by pysel**
Closes: #5227 

## What is the purpose of the change

Adds labels to dependabot to avoid adding them manually 

## Testing and Verifying

Minor 

## Documentation and Release Note

Changelog

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -5,10 +5,16 @@ updates:
...
+  labels:
+    - "A:no-changelog"
+    - "V:state/compatible/no_backport"
 - package-ecosystem: gomod
   directory: "/"
   schedule:
     interval: weekly
+  labels:
+    - "A:no-changelog"
+    - "V:state/compatible/no_backport"
```
Comment (pysel) [inline .github/dependabot.yml:17]: as of my knowledge, it is not possible to add global labels, hence, here I have to duplicate labels from the previous `package-ecosystem`

Comment (ValarDragon) [reply .github/dependabot.yml:17]: I don't think we should be doing this for the go mod updates, they can be state breaking

Comment (pysel) [reply .github/dependabot.yml:17]: hm, I thought that the reviewer would change the label if the change is breaking, but I yeah your point makes sense, it is just better if we do not have labels for potentially breaking changes at all

File: CHANGELOG.md [hunk 2]
```diff
@@ -70,6 +70,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   * [#5165](https://github.com/osmosis-labs/osmosis/pull/5165) Improve error message when fully exiting from a pool.
   * [#5187](https://github.com/osmosis-labs/osmosis/pull/5187) Expand `IncentivizedPools` query to include internally incentivized CL pools.
   * [#5239](https://github.com/osmosis-labs/osmosis/pull/5239) Implement `GetTotalPoolShares` public keeper function for GAMM.
+  * [#5250](https://github.com/osmosis-labs/osmosis/pull/5250) Add changelog and state compatibility labels to dependabot.
```
Comment (mattverse) [inline CHANGELOG.md:73]:
```suggestion
```

**Pull request merged**